doc/api/fs.md

```diff
@@ -2952,6 +3012,35 @@ The Node.js GitHub issue [#25741][] provides more information and a detailed
 analysis on the performance of `fs.readFile()` for multiple file sizes in
 different Node.js versions.
 
+### `fs.readJSON(path[, options], callback)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `path` {string|Buffer|URL|integer} filename or file descriptor
+* `options` {Object|string}
+  * `encoding` {string|null} **Default:** `'utf-8'`
```

Comment:
The use of `'utf8'` VS `'utf-8'` is more prominent in the docs. Anyway it would be nice if it was consistent but that's a work for another time.

Reply:
https://github.com/nodejs/node/pull/37945